### DISCUSSION TITLE
Add Spatial Audio to IINA

### DISCUSSION THREAD

**Discussion by aliraoufian** [iina/iina#5182] [Ideas]
Add Spatial Audio Converter in IINA for better sound with Apple AirPods

**Comment by low-batt** [comment]
This has already been requested in issue #3444. Briefly, IINA is currently under heavy development for a feature release. As a part of this `libmpv` is being upgraded to 0.38.0. The update adds a new [AVFoundation](https://developer.apple.com/av-foundation/) audio driver which supports Spatial Audio. In the development version of IINA the audio settings have been updated to allow you to select this driver as shown in [this comment](https://github.com/iina/iina/issues/3444#issuecomment-2301477510).

**Reply by fsafq41** [reply]
Thanks for finally bring Spatial Audio to IINA!
Tried it on the latest nightly (1.3.5 Build 141, Built on Sat Dec 14 2024) and it works well for Fixed.
Head Tracked doesn't work though (it behaves the same as Fixed).

**Reply by low-batt** [reply]
I tested playing the YouTube video [Stereo Test - Left/Right Audio Test for Headphones/Speakers](https://www.youtube.com/watch?v=YwNs1Z0qRY0) with the IINA Nightly build. Turning my head left during the very first part of that video moved the sound away from the left speaker.

It is possible to turn off head tracking. When you click on `Sound` in `Control Center` while playing does it show `Head Tracked` checked as seen here?:
![audio](https://github.com/user-attachments/assets/a5f483ab-f3e1-44d6-b2b2-8721141077ac)

**Reply by lucifercraig12345-create** [reply]
Hii. I wanted to know if I can use IINA to liste a video file in Spatial Audio using AVFoundation on my wired apple earpods.
